recipes/python-toolchain/bld.bat

```diff
@@ -0,0 +1,19 @@
+:: This works for `setuptools`, but breaks `distutils`.
+:: Will need to figure out a better long term strategy.
+copy "%RECIPE_DIR%\distutils.cfg" "%STDLIB_DIR%\distutils\distutils.cfg"
+if errorlevel 1 exit 1
+
+:: Configure `pip`.
+IF NOT EXIST "%LIBRARY_PREFIX%\etc" mkdir "%LIBRARY_PREFIX%\etc"
+if errorlevel 1 exit 1
+copy "%RECIPE_DIR%\pip.conf" "%LIBRARY_PREFIX%\etc\pip.conf"
+if errorlevel 1 exit 1
```

Comment:
no idea:

The only packages I have here which install etc directories are 

`m2-filesystem-2016.04-3\Library\etc\` -> all kind of bash stuff
`go-1.4.2-0\etc\conda\` -> The activate.d stuff...
 

I also have
* `<env>\share` -> jupyter stuff, like `...\share\jupyter\kernels\python3` and some man page in `..\share\man\man1\...`
* `<env>\Skripts` for the binaries which should become available in PATH

and

* `<env>\Library\bin` (which is also in PATH), but seems to only have qt stuff (designer, etc) and some dlls in it
* `<env>\Library\share\man\` -> libpng man pages
* `<env>\Library\share\pkgconfig`

So basically it seems to be "undefined" :-)